apps/settings/elements/languages.html

```diff
@@ -29,6 +29,11 @@ <h1 data-l10n-id="language-header"> Language </h1>
             </select>
           </span>
         </li>
+        <li>
+          <a class="menu-item" id="more-languages" href="#">
```

Comment:
There are many panels in settings app and they all in the same DOM tree. Using class names avoids potential name collisions.